security-manager/src/main/resources/schema/wildfly-security-manager_3_0.xsd

```diff
@@ -1,113 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-
-<!--
-  ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2011, Red Hat, Inc., and individual contributors
-  ~ as indicated by the @author tags. See the copyright.txt file in the
-  ~ distribution for a full listing of individual contributors.
-  ~
-  ~ This is free software; you can redistribute it and/or modify it
-  ~ under the terms of the GNU Lesser General Public License as
-  ~ published by the Free Software Foundation; either version 2.1 of
-  ~ the License, or (at your option) any later version.
-  ~
-  ~ This software is distributed in the hope that it will be useful,
-  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
-  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
-  ~ Lesser General Public License for more details.
-  ~
-  ~ You should have received a copy of the GNU Lesser General Public
-  ~ License along with this software; if not, write to the Free
-  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
-  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
-  -->
-
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
-           targetNamespace="urn:jboss:domain:security-manager:3.0"
-           xmlns="urn:jboss:domain:security-manager:3.0"
-           elementFormDefault="qualified"
-           attributeFormDefault="unqualified"
-           version="1.0">
-
-    <!-- The security manager subsystem root element -->
-    <xs:element name="subsystem" type="securityManagerSubsystemType"/>
-
-    <xs:complexType name="securityManagerSubsystemType">
-        <xs:annotation>
-            <xs:documentation>
-                <![CDATA[
-                The security manager subsystem configuration.
-                        ]]>
-            </xs:documentation>
-        </xs:annotation>
-        <xs:sequence>
-            <xs:element name="deployment-permissions" minOccurs="0" maxOccurs="1" type="deploymentPermissionsConfigType"/>
-        </xs:sequence>
-    </xs:complexType>
-
-    <xs:complexType name="deploymentPermissionsConfigType">
-        <xs:annotation>
-            <xs:documentation>
-                <![CDATA[
-                The deploymentPermissionConfigType specifies the elements and attributes that can be used to configure
```

Comment:
pure removal of wildfly-security-manager_3_0.xsd is not the right thing to me.

`<xs:complexType name="deploymentPermissionsConfigType">` and `<xs:complexType name="permissionType">` has better `<xs:documentation>` in XSD version 3.0 comparing to 1.0

Thus I would suggest to also enhance XSD version 1.0 with the diff as part of this PR